src/bin/adbackup.rs

```diff
@@ -20,4 +44,27 @@ fn make_clap<'a, 'b>() -> clap::App<'a, 'b> {
         .author(crate_authors!())
         .version(adbackup::version())
         .setting(AppSettings::SubcommandRequiredElseHelp)
+        .arg(Arg::with_name("verbose")
+            .short("v")
+            .multiple(true)
+            .global(true)
+            .help("Increases logging verbosity each use for up to 3 times"))
+        .subcommand(SubCommand::with_name("devices")
+            .about("List connected devices")
+            .help("List all android devices connected to your pc with enabled debug mode."))
+}
+
+fn print_devices() -> Result<(), Error> {
+    let devices = Device::list_devices()?;
+
+    if devices.len() > 0 {
+        info!("Found the following devices:");
+        devices.into_iter().for_each(|device|
+            info!("Id: '{}', Name: '{}'", device.id, device.name))
+    } else {
+        warn!("No device found. Make sure that you connect at least one device with enabled \
```

Comment:
is this really a warning? I think warnings should be issued if something went wrong but not having a device attached is a normal use case, isn't it?

Reply:
😂 thought about the same thing quiet a while. My conclusion was that if I wan't to list my connected devices I assume that at least one device is connected and from my point of view it is not just an info if there is none...you would prefer to show just an info message?

Reply:
> you would prefer to show an info message?

I think I just got it... warnings are the next verbosity state after infos, right? So while not running the tool with -v the `list_devices` will not print anything instead of the warning? If so then that is totally fine but requires a follow-up question on my part for #9. will post it there